src/target/riscv/riscv.c

```diff
@@ -4505,10 +4515,19 @@ COMMAND_HANDLER(riscv_set_enable_eq_match_trigger)
 	RISCV_INFO(r);
 
 	if (CMD_ARGC == 0) {
-		command_print(CMD, "equality match trigger enabled: %s", r->enable_equality_match_trigger ? "on" : "off");
+		command_print(CMD, "enable equality match trigger: for wp (%s)",
+			      r->wp_enable_equality_match_trigger ? "on" : "off");
 		return ERROR_OK;
 	} else if (CMD_ARGC == 1) {
-		COMMAND_PARSE_ON_OFF(CMD_ARGV[0], r->enable_equality_match_trigger);
+		if (!strcmp(CMD_ARGV[0], "wp")) {
+			r->wp_enable_equality_match_trigger = true;
+		} else if (!strcmp(CMD_ARGV[0], "none")) {
+			r->wp_enable_equality_match_trigger = false;
+		} else {
+			LOG_ERROR("Command takes 1 of the following parameters: wp, none");
```

Comment:
Please, remove the `LOG_ERROR(...)`.
As Antonio [mentioned here](https://review.openocd.org/c/openocd/+/8071/comment/7b456479_aabd7afb/) 
> When a command throws an ERROR_COMMAND_SYNTAX_ERROR, all it's previous output is erased and replaced by the output of 'usage command_name'. So there is no need to put any command_print() before return ERROR_COMMAND_SYNTAX_ERROR, as it would be dropped. And no need for LOG_ERROR() to report the syntax, as it would be already printed by 'usage ..'.

Reply:
Addressed